Add note about HSMs (#1232)
tableau/connector-plugin-sdk: docs/package-sign.md

### Signing example

> Getting a certificate is a multi-step process. This example illustrates how to sign a TACO file with a basic signed certificate. Before starting, read the documentation for your certificate authority to make sure youa re complying with any additional requiremnts. 

Reviewer: ```suggestion
Getting a certificate is a multi-step process. This example illustrates how to sign a TACO file with a basic signed certificate. Before starting, read the documentation for your certificate authority to make sure you are complying with any additional requirements. 
```
